src/site/content/en/blog/indexeddb/index.md

```diff
@@ -228,196 +287,293 @@ All data operations in IndexedDB are carried out inside a transaction. Each oper
+To update an existing item in the object store, use the `put()` method on an object containing the
+same primary key value as the object in the store. Here, the assumption is that the `keyPath` for
+the store object store is the `"name"` property and that you are updating the price and description
+of the `"sandwich"` object. The database interaction has the same structure as the create and read
+operations: get the database object, create a transaction, open an object store on the transaction,
+and perform the operation on the object store.
 
 ### Deleting data
 
-To delete data, call the  [`delete`](https://developer.mozilla.org/docs/Web/API/IDBObjectStore/delete) method on the object store.
+To delete data, call the
+[`delete()`](https://developer.mozilla.org/docs/Web/API/IDBObjectStore/delete) method on the object
+store.
 
 ```js
 someObjectStore.delete(primaryKey);
 ```
 
-Once again, this method returns a promise and must be wrapped in a transaction. Here is a simple example:
+Once again, this method returns a promise and must be wrapped in a transaction. Here is a simple
+example:
 
 ```js
-dbPromise.then(function(db) {
-  var tx = db.transaction('store', 'readwrite');
-  var store = tx.objectStore('store');
-  store.delete(key);
-  return tx.complete;
-}).then(function() {
-  console.log('Item deleted');
-});
+dbPromise
+  .then(function (db) {
+    const tx = db.transaction('store', 'readwrite');
+    const store = tx.objectStore('store');
+    store.delete(key);
+    return tx.complete;
+  })
+  .then(function () {
+    console.log('Item deleted.');
+  });
 ```
 
-The structure of the database interaction is the same as for the other operations. Note that we again check that the whole transaction has completed by returning the `tx.complete` method to be sure that the delete was carried out.
+The structure of the database interaction is the same as for the other operations. Note that you
+again check that the whole transaction has completed by returning the `tx.complete()` method to be
+sure that the deletion was carried out.
 
 ## Getting all the data {: #get }
 
-So far we have only retrieved objects from the store one at a time. We can also retrieve all of the data (or subset) from an object store or index using either the `getAll` method or using cursors.
+So far you have only retrieved objects from the store one at a time. You can also retrieve all of
+the data (or a subset) from an object store or index using either the `getAll()` method or using
+cursors.
 
-### Using the getAll method
+### Using the `getAll()` method
```

Comment:
```suggestion
### Using the getAll() method
```

Reply:
Not in headings. 

The reason for marking this way in text is not calling out code, per se. The traditional English way of calling out something extra in text would have been to put quotation marks around it. This could be misinterpreted as part of the code in many situations. Using special formatting is merely about avoiding confusion. 

Avoiding special formatting in headings is about aesthetics; it avoids ransom note look.